views/page.html.pug

```diff
@@ -21,3 +21,7 @@ html
           div(class="my-6") !{post.message}
           each image in post.images
             img(class="min-w-full max-w-full h-auto" src=image alt="")
+          if post.link != null
```

Comment:
Minor: I would use `!post_link` (if it doesn't need to be strict check) or `post.link !== null` (if it needs to be strict null check).